syntax = "proto3";

message PmioState {
  uint32  pm1_status      = 1;
  uint32  pm1_enable      = 2;
  uint32  pm1_control     = 3;
  uint32  pm1_timer       = 4;

  uint64  gpe0_status     = 10;
  uint64  gpe0_enable     = 11;

  uint32  smi_enable      = 12;
  uint32  smi_status      = 13;
}
